main.py

```
@@ -273,43 +301,386 @@ def event_message(event,replyToken,userId,user_data):
     res = []
     mesType = event['message']['type']
     if mesType == "text":
-        res = message_process(event,userId,user_data)
+        res = message_process(event,userId,user_data,replyToken)
     else:
         res = ["テキストメッセージ以外には対応していません"]
-    la.reply_to_line(LINE_ACCESS_TOKEN, replyToken, res)
+    try:
+        la.reply_to_line(LINE_ACCESS_TOKEN, replyToken, res)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)